[native] Prestissimo worker metrics documentation (#23107)
prestodb/presto: presto-docs/src/main/sphinx/presto_cpp/properties.rst

  
  The exceeding capacity must allocate from the non-reserved query memory.

``runtime-metrics-collection-enabled``
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
* **Type:** ``boolean``
* **Default value:** ``false``

>  Enables collection of worker level metrics. If the Prestissimo server was compiled with

Reviewer: ```suggestion
  Enables collection of worker level metrics. If the Presto C++ server was compiled with
```

>  PRESTO_ENABLE_PROMETHEUS_REPORTER set to ON, the server exposes an endpoint ``/v1/info/metrics`` to collect

Reviewer: ```suggestion
  PRESTO_ENABLE_PROMETHEUS_REPORTER set to ON, the server exposes an endpoint ``/v1/info/metrics`` to collect
```

>  metrics in prometheus format.

Reviewer: ```suggestion
  metrics in Prometheus format.
```
